its/src/test/java/com/sonar/it/scanner/msbuild/sonarqube/CodeCoverageTest.java

```diff
@@ -44,58 +41,49 @@
 import static org.assertj.core.api.Assertions.tuple;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
-@ExtendWith(ServerTests.class)
+@ExtendWith({ServerTests.class, ContextExtension.class})
 class CodeCoverageTest {
   @TempDir
   public Path basePath;
```

Comment:
Do you plan to remove this instance as well? If so, is there a follow-up ticket to track it?